Readme.md

```diff
@@ -97,6 +97,12 @@ Use `inlineCss: true` to enable this feature.
 
 TODO: as soon as the feature will be stable it should be enabled by default.
 
+### puppeteerExecutablePath
```

Comment:
Name section `Docker` and move under `Containers and other restricted environments`.

Reply:
Reasonable.